rllib/utils/test_utils.py

```diff
@@ -33,6 +34,14 @@
 logger = logging.getLogger(__name__)
 
 
+def add_gpu_if_necessary(config: "TrainerConfig"):
+    num_gpus = int(os.environ.get("RLLIB_NUM_GPUS", 0))
```

Comment:
Wait, I feel like if `RLLIB_NUM_GPUS` is NOT defined, then we shouldn't do anything with the config, just leave as-is. With this logic here, we set `num_gpus=0`, which is counterintuitive. There could be test cases, where you would want to use a GPU ("hard-coded" in the test case's config), but then it gets set to num_gpus=0, b/c the env variable is not defined.

Reply:
This is fair. That's why I had it as a util function in the first place so that I only change it once for all the algos.